app/javascript/mastodon/features/status/index.jsx

```diff
@@ -637,6 +646,10 @@ class Status extends ImmutablePureComponent {
     const isLocal = status.getIn(['account', 'acct'], '').indexOf('@') === -1;
     const isIndexable = !status.getIn(['account', 'noindex']);
 
+    if (!isLocal) {
+      remoteHint = <RemoteHint url={status.get('url')} />
```

Comment:
```suggestion
      remoteHint = <TimelineHint url={status.get('url')} resource={<FormattedMessage id='timeline_hint.resources.replies' defaultMessage='Some replies' />} />
```

(Note: you may need to reformat this via the linting)